### ISSUE TITLE
Incorrect handling of excluded sub directories

### ISSUE THREAD

**Issue by mortalapeman** [cldwalker/smart-ignore#2]
When adding `js/` to a `.gitignore` file, the behavior of git is to ignore all directories with the name `js`. The current regex generated is `#"js/"` which causes light table to exclude directories like `src-cljs/`. To replicate git's behavior, all directory exclusion entries should be prefixed with `^` to ensure they only match directories with that exact name.
### Steps to reproduce:
1. Setup test directory:
   
   ```
   mkdir smart-git-test
   cd smart-git-test
   echo "js/" > .gitignore
   mkdir root root/js root/src-cljs
   touch root/js/should.be.hidden.txt
   touch root/src-cljs/should.not.be.hidden.txt
   ```
2. Open Light Table and verify that the file should.not.be.hidden.txt is hidden.

**Comment by cldwalker** [comment]
Thanks for reporting your issue! This is [one of my 14 active issues](https://gh-active-issues.herokuapp.com/#cldwalker/smart-ignore#2). Use that link to check how soon your issue will be answered. Don't forget to check your issue against this project's CONTRIBUTING.md. Cheers.

**Comment by cldwalker** [comment]
Thanks for the detailed report! Should be fixed in 0.1.1

**Comment by mortalapeman** [comment]
Awesome, thanks for the quick turn around.
